ibridges/cli/data_operations.py

```diff
@@ -139,6 +139,12 @@ def _mod_parser(cls, parser):
             type=Path,
             nargs="?",
         )
+        parser.add_argument(
+            "--on-error",
+            help="'fail' (default): stop on error; 'warn': warn and continue; 'skip': continue",
```

Comment:
Perhaps this help section can be expanded a little bit? Like "When iBridges encounters any error during the process of downloading/uploading/syncing, .."